File: Java/01.Java-Basic/04.ConditionalStatementsExercise/src/WorldSwimmingRecord.java
import java.util.Scanner;

public class WorldSwimmingRecord {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double record = Double.parseDouble(scanner.nextLine());
        double distance = Double.parseDouble(scanner.nextLine());
        double secondsPerMeter = Double.parseDouble(scanner.nextLine());

        double c1 = distance * secondsPerMeter;
        double c2 = Math.floor(distance / 15) * 12.5;
        double current = c1 + c2;
        if (current < record) {
            System.out.printf("Yes, he succeeded! The new world record is %.2f seconds.", current);
        } else {
            System.out.printf("No, he failed! He was %.2f seconds slower.", current-record);
        }
    }
}

File: JavaScript/01.JavaScriptBasic/04Conditional-Statements-Exercise/src/WorldSwimmingRecord.js
function worldRecord(input) {
    let recordTime = Number(input[0]);
    let distance = Number(input[1]);
    let timePerMeter = Number(input[2]);

    let time = distance * timePerMeter;
    let extraTime = (Math.floor(distance / 15)) * 12.5;
    time += extraTime;
    if (time < recordTime) {
        console.log(`Yes, he succeeded! The new world record is ${time.toFixed(2)} seconds.`);
    } else {
        console.log(`No, he failed! He was ${Math.abs(recordTime - time).toFixed(2)} seconds slower.`)
    }
}

worldRecord(["10464", "1500", "20"]);
worldRecord(["55555.67", "3017", "5.03"]);
